docs/modules/ROOT/partials/apis/camel-k-crds.adoc

```diff
@@ -8240,8 +8240,6 @@ Deprecated: for backward compatibility.
 * <<#_camel_apache_org_v1_IntegrationKitTraits, IntegrationKitTraits>>
 * <<#_camel_apache_org_v1_Traits, Traits>>
 
-WARNING: The Registry trait is **deprecated** and will removed in future release versions.
```

Comment:
Probably you need to amend the crds doc generation as well to include the deprecation notice in the API description

Reply:
Oh dear, this seems to be using github.com/tadayosi/gen-crd-api-reference-docs@v0.4.0-camel-k-2
Do we really need to bring information from the badges to the CRD docs? How can this possibly be done without creating a maintenance nightmare?

Reply:
We can probably keep both image and description message, I don't think they are incompatible.